app/models/skipped_email_log.rb

```diff
@@ -39,6 +39,9 @@ def self.reason_types
       user_email_access_denied: 22,
       sender_topic_deleted: 23,
       user_email_no_email: 24,
+      group_smtp_post_deleted: 25,
+      group_smtp_topic_deleted: 25,
+      group_smtp_disabled_for_group: 25,
       # you need to add the reason in server.en.yml below the "skipped_email_log" key
```

Comment:
Something doesn't seem right here, the integers should be different between the keys.

Reply:
Damn it, silly copy paste mistake, sorry.